boilerplate/package.json

```diff
@@ -78,36 +80,26 @@
     "eslint-plugin-promise": "4.2.1",
     "eslint-plugin-react": "7.21.5",
     "eslint-plugin-react-native": "3.10.0",
+    "expo-modules-autolinking": "^0.3.4",
```

Comment:
Just curious: AFAIK  `expo` script does autolinking, what does this dependency do?

Reply:
It's a dependency of `expo`, but for some reason I had to make certain packages explicit or pnpm wouldn't set them up properly. Weird, I nkow.